modules/twinklefluff.js

```diff
@@ -29,173 +36,174 @@ Twinkle.fluff = {
+			var vandLink = document.createElement('a');
+			var normLink = document.createElement('a');
 
-				var agfLink = document.createElement('a');
-				var vandLink = document.createElement('a');
-				var normLink = document.createElement('a');
+			agfLink.href = "#";
+			vandLink.href = "#";
+			normLink.href = "#";
+			$(agfLink).click(function(){
+				Twinkle.fluff.revert('agf', vandal);
+			});
+			$(vandLink).click(function(){
+				Twinkle.fluff.revert('vand', vandal);
+			});
+			$(normLink).click(function(){
+				Twinkle.fluff.revert('norm', vandal);
+			});
 
-				agfLink.href = "#";
-				vandLink.href = "#";
-				normLink.href = "#";
-				$(agfLink).click(function(){
-					Twinkle.fluff.revert('agf', vandal);
-				});
-				$(vandLink).click(function(){
-					Twinkle.fluff.revert('vand', vandal);
-				});
-				$(normLink).click(function(){
-					Twinkle.fluff.revert('norm', vandal);
-				});
+			agfLink.appendChild( spanTag( 'Black', '[' ) );
+			agfLink.appendChild( spanTag( 'DarkOliveGreen', 'rollback (AGF)' ) );
+			agfLink.appendChild( spanTag( 'Black', ']' ) );
 
-				agfLink.appendChild( spanTag( 'Black', '[' ) );
-				agfLink.appendChild( spanTag( 'DarkOliveGreen', 'rollback (AGF)' ) );
-				agfLink.appendChild( spanTag( 'Black', ']' ) );
+			vandLink.appendChild( spanTag( 'Black', '[' ) );
+			vandLink.appendChild( spanTag( 'Red', 'rollback (VANDAL)' ) );
+			vandLink.appendChild( spanTag( 'Black', ']' ) );
 
-				vandLink.appendChild( spanTag( 'Black', '[' ) );
-				vandLink.appendChild( spanTag( 'Red', 'rollback (VANDAL)' ) );
-				vandLink.appendChild( spanTag( 'Black', ']' ) );
+			normLink.appendChild( spanTag( 'Black', '[' ) );
+			normLink.appendChild( spanTag( 'SteelBlue', 'rollback' ) );
+			normLink.appendChild( spanTag( 'Black', ']' ) );
 
-				normLink.appendChild( spanTag( 'Black', '[' ) );
-				normLink.appendChild( spanTag( 'SteelBlue', 'rollback' ) );
-				normLink.appendChild( spanTag( 'Black', ']' ) );
+			agfNode.appendChild(agfLink);
+			vandNode.appendChild(vandLink);
+			normNode.appendChild(normLink);
 
-				agfNode.appendChild(agfLink);
-				vandNode.appendChild(vandLink);
-				normNode.appendChild(normLink);
+			revertNode.appendChild( agfNode );
+			revertNode.appendChild( document.createTextNode(' || ') );
+			revertNode.appendChild( normNode );
+			revertNode.appendChild( document.createTextNode(' || ') );
+			revertNode.appendChild( vandNode );
 
-				revertNode.appendChild( agfNode );
-				revertNode.appendChild( document.createTextNode(' || ') );
-				revertNode.appendChild( normNode );
-				revertNode.appendChild( document.createTextNode(' || ') );
-				revertNode.appendChild( vandNode );
+			ntitle.insertBefore( revertNode, ntitle.firstChild );
+		}
+	},
 
-				ntitle.insertBefore( revertNode, ntitle.firstChild );
-			}
+	normal: function() {
+		if( mw.config.get('wgNamespaceNumber') === -1 && mw.config.get('wgCanonicalSpecialPageName') === "Contributions" ) {
+			this.contributions();
```

Comment:
Did you test this? I'm pretty sure this isn't called from an object context, so `this` will be undefined.

Reply:
Replacing `this` with `Twinkle.fluff` here and below should be sufficient, no?  A quick test suggests this works, @atlight @MusikAnimal were there any other issues either of you recall?

If so, I think we can handle the merge conflict on-wiki if @WMDE-Fisch isn't around

Reply:
Yeah, this works - enabled on test.wiki